README.md

```diff
@@ -1,138 +1,132 @@
-# [![The Buf logo](.github/buf-logo.svg)][buf] protovalidate-java
+[![The Buf logo](.github/buf-logo.svg)][buf] 
+
+# protovalidate-java
```

Comment:
At some point we should add some more badges to this README including the latest version number and potentially a link to published Javadocs.